TestCases/compliance-level-3/0055-feel-odd-function/0055-feel-odd-function.dmn

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<definitions namespace="http://www.montera.com.au/spec/DMN/0055-feel-odd-function" name="0055-feel-odd-function" id="_i9fboPUUEeesLuP4RHs4vA" xmlns="http://www.omg.org/spec/DMN/20180521/MODEL/" xmlns:di="http://www.omg.org/spec/DMN/20180521/DI/" xmlns:dmndi="http://www.omg.org/spec/DMN/20180521/DMNDI/" xmlns:dc="http://www.omg.org/spec/DMN/20180521/DC/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <description>FEEL built-in function 'odd(number)' in category numeric functions</description>
+    <decision name="decision001" id="_decision001">
+        <description>Tests FEEL expression: 'odd(2)' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'odd(2)'?</question>
+        <allowedAnswers>false  (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision001"/>
+        <literalExpression>
+            <text>odd(2)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision001_a" id="_decision001_a">
+        <description>Tests FEEL expression: 'odd(1)' and expects result: 'true (boolean)'</description>
+        <question>Result of FEEL expression 'odd(1)'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision001_a"/>
+        <literalExpression>
+            <text>odd(1)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision002" id="_decision002">
+        <description>Tests FEEL expression: 'odd(-2)' and expects result: 'false (boolean)'</description>
+        <question>Result of FEEL expression 'odd(-2)'?</question>
+        <allowedAnswers>false (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision002"/>
+        <literalExpression>
+            <text>odd(-2)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision002_a" id="_decision002_a">
+        <description>Tests FEEL expression: 'odd(-1)' and expects result: 'true (number)'</description>
+        <question>Result of FEEL expression 'odd(-1)'?</question>
+        <allowedAnswers>true (boolean)</allowedAnswers>
+        <variable typeRef="boolean" name="decision002_a"/>
+        <literalExpression>
+            <text>odd(-1)</text>
+        </literalExpression>
+    </decision>
+    <decision name="decision003" id="_decision003">
+        <description>Tests FEEL expression: 'odd(0)' and expects result: '0 (number)'</description>
+        <question>Result of FEEL expression 'odd(0)'?</question>
+        <allowedAnswers>0 (number)</allowedAnswers>
```

Comment:
from ` <allowedAnswers>0 (number)</allowedAnswers>`
to ` <allowedAnswers>false (boolean)</allowedAnswers>`

Reply:
@tarilabs amended.